socketEvent.js

```diff
@@ -0,0 +1,189 @@
+const {
+  request,
+  io,
+  raceRooms,
+  syncraceget,
+  paraTypedByUser,
+  expiryInSeconds,
+  TFSBackendURL,
+} = require("./utils.js");
+
+exports.disconnect = function (socket) {
+  socket.on("disconnect", () => {
```

Comment:
Make event names capital and store them in constants file.

Reply:
That's a built-in event by socket